Documentation/MyFirstContribution.txt

```diff
@@ -963,7 +962,7 @@ commenting on your PR with `/submit`.
 [[responding-ggg]]
 === Updating With Comments
 
-Skip ahead to <<reviewing,Responding to Reviews>> for information on how to
+Skip ahead to <<reviewing, Responding to Reviews>> for information on how to
```

Comment:
There's no need to add a space here, only the "Responding to Reviews" part will be visible in the rendered document.

Reply:
well, thank you for improving it further

Reply:
well thank you for your review , I'll look after it

Reply:
well thank you for your review , I'll look after it

Reply:
well thank you for your review, I'll look into it